ogticrd/covid-safe-paths
# Fix: Changed resource name

#### Description:


Changed wording name of the button placed on FAQs screen
#### Linked issues:


Fix: [184](https://trello.com/c/qnZ1e4ca/184-cambiar-wording-en-el-bot%C3%B3n-inferior-de-faqs)
#### Screenshots:


![image](https://user-images.githubusercontent.com/43874147/86067351-b249a480-ba42-11ea-8b30-4a48bfe39974.png)

#### How to test:


- `git pull && git checkout Fix/184-change-button-wording`
- Head over to FAQs screen
- Check the wording name of the button placed in the bottom of the screen
